### DISCUSSION TITLE
Awsome program! - possible contradiction in your frontpage help

### DISCUSSION THREAD

**Discussion by nOrphf** [nref/BrowseRouter#21] [General]
Hi

I have just learned about this magnificent piece of software. Simple to use, but gives so many possibilities that my mind is so blown away, that I can't figure out how to start using it :D

I don't know if I have misunderstood some of you help text, but in you config file you write:
"; Applied only if no url preference match is found."
But int you explanation of "Sources" you write:
"In the case of a conflict between a source preference and a URL preference, the source preference wins."

isn't that a contradiction? - I'm at least confused about which one wins? :)

Regards Lars

**Comment by nref** [comment]
I appreciate the enthusiasm! BR is a pretty simple little program. Let me know what uses you come up with!

You're right, the text "Applied only if no url preference match is found" was incorrect. I just now updated the README. Thanks!
